packages/config/maintenance/lintConfigFiles.ts

```diff
@@ -604,50 +608,51 @@ Consider converting this parameter to unsigned using ${white(
 				for (const [key, param] of partialParams) {
 					const bitMask = key.valueBitMask!;
 					const shiftAmount = getMinimumShiftForBitMask(bitMask);
+					const bitMaskWidth = getBitMaskWidth(bitMask);
 					const shiftedBitMask = bitMask >>> shiftAmount;
-					// TODO: Find out how to test this with negative values
-					if (
-						param.minValue >= 0 &&
-						(param.minValue & shiftedBitMask) !== param.minValue
-					) {
+					const minValue =
+						param.unsigned || bitMaskWidth == 1
+							? 0
+							: -(1 << bitMaskWidth) / 2;
+					const maxValue =
+						param.unsigned || bitMaskWidth == 1
+							? (1 << bitMaskWidth) - 1
+							: (1 << bitMaskWidth) / 2 - 1;
```

Comment:
with `0xFF`, `bitMaskWidth` will be 8, `-(1 << 8) / 2` gives us `minValue = -128` and `(1 << 8) / 2 - 1` gives us `maxValue = 127` for the signed params.